src/inmanta/moduletool.py

```diff
@@ -683,10 +705,10 @@ def modules_parser_config(cls, parser: ArgumentParser) -> None:
   version.
 When a development release is done using the --dev option, this command:
 * Does a commit that updates the current version of the module to a development version that is a patch, minor or major version
-  ahead of the previous stable release. The size of the increment is determined by the --patch, --minor or --major argument
-  (--patch is the default). When a CHANGELOG.md file is present in the root of the module directory then the version number in
-  the changelog is also updated accordingly. The changelog file is always populated with the associated stable version and
-  not a development version.
+  ahead of the previous stable release. The size of the increment is determined by the --revision (only with 4 digits version),
```

Comment:
I would allow this for 3-digit versions as well. I see no reason not to, do you?